build/includes/website.mk

```diff
@@ -24,6 +24,23 @@
 # Website targets
 #
 
+REL_PATH := content/en/docs/Reference/agones_crd_api_reference.html
+API_DOC_PATH := "$(mount_path)/site/$(REL_PATH)"
+GEN_API_DOCS ?= docker run -e FILE=$(API_DOC_PATH) --rm -i $(common_mounts) $(build_tag) bash -c "/go/src/agones.dev/agones/site/gen-api-docs.sh"
```

Comment:
Curious as to the motivation to put all of this in variables? Seems like it wouldn't change?

Reply:
I have added those options to be able to split test html result and normal one. GEN_API_DOCS used twice, that's why it was extracted.